tests/results/TextPositionTest.py

```diff
@@ -30,3 +40,58 @@ def test_properties(self):
         uut = TextPosition(8, 39)
         self.assertEqual(uut.line, 8)
         self.assertEqual(uut.column, 39)
+
+
+class TextPositionLineColTest(unittest.TestCase):
+
+    def test_from_position(self):
```

Comment:
yeah but since it belongs to the same function i wouldn't mind if i don't get further tests on a faulty function?, i'm not saying that's always the case, but all of the tests here are very specific so it would break everything if even one fails :P